File: Java Script - Introduction to Programming/8.1. Exam Preparation - Part I/05. Date after 5 Days.js
function dateAfter5Days([...arg]) {
  let [day, month] = [...arg.map(Number)]
  const daysObj = {1: 31, 2: 28, 3: 31, 4: 30, 5: 31, 6: 30, 7: 31, 8: 31, 9: 30, 10: 31, 11: 30, 12: 31}
  
  day += 5

  if (daysObj[month] < day ) {
    day -= daysObj[month] 
    month++
    month > 12 ? month -= 12 : null 
  }

  month < 10 ? month = '0' + month : null 
  return `${day}.${month}`
}

// console.log(dateAfter5Days(['28', '03']))
// console.log(dateAfter5Days(['27', '12']))
// console.log(dateAfter5Days(['25', '1']))
// console.log(dateAfter5Days(['26', '02']))

File: Pyrhon - Introduction to Programming/8.1. Exam Preparation - Part I/05. Date after 5 Days.py
day, month = int(input()), int(input())
days_dict = {1: 31, 2: 28, 3: 31, 4: 30, 5: 31, 6: 30, 7: 31, 8: 31, 9: 30, 10: 31, 11: 30, 12: 31}
  
day += 5

if days_dict[month] < day:
    day -= days_dict[month]
    month+= 1
    if month > 12:
        month -= 12

if month < 10:
  month = '0' + str(month)

print(f'{day}.{month}')



'''
28
03
#2.04
'''
